provider/core/src/response.rs

```diff
@@ -570,6 +570,28 @@ where
         })
     }
 
+    /// Convert between two [`DataMarker`] types that are compatible with each other.
+    ///
+    /// This happens if they both have the same [`DataMarker::Yokeable`] type.
+    ///
+    /// Can be used to erase the key of a data payload in cases where multiple keys correspond
```

Comment:
nit:

```suggestion
    /// Can be used to type-erase the key of a data payload in cases where multiple keys correspond
```

Reply:
Mmm, we're not erasing the type. We are actually erasing the key, by downgrading from a KeyedDataMarker to a DataMarker.